core/loadpoint.go

```diff
@@ -110,6 +111,7 @@ type LoadPoint struct {
 	MinCurrent    float64       // PV mode: start current	Min+PV mode: min current
 	MaxCurrent    float64       // Max allowed current. Physically ensured by the charger
 	GuardDuration time.Duration // charger enable/disable minimum holding time
+	Circuits      []*Circuit    // list of circuits this lp belongs to
```

Comment:
Das brauchst du weil circuits keine Hierarchie bilden? Schöner wäre wenn ein LP nur zu einem Circuit gehören könnte?

Reply:
LP können in mehreren Kreisen sein. Selbst wenn es Hierarchien gibt, muss der LP für jeden Kreis geprüft werden. 
Aktuell wäre es der Kreis der Site und ggf ein Unterkreis. Circuits sind aktuell nicht hierarchisch, aber ein LP kann in mehreren Kreisen sein